src/zope/app/apidoc/codemodule/browser/menu.pt

```diff
@@ -28,10 +29,10 @@
          tal:attributes="href info/url"
          tal:content="info/path">
         /zope/app/Application
-      </a>
+      </a><br />
```

Comment:
You're adding one `<br />` after all the repeated `<a>` and I don't think that was your intention.